_tools/redirects/redirects.csv

```diff
@@ -324,7 +324,13 @@ source,destination
 /reference/shading_language.html,/learning/features/shading/shading_language.html
 /reference/unity_to_godot.html,/learning/editor/unity_to_godot.html
 /reference/variant_class.html,/development/cpp/variant_class.html
+introduction_to_global_illumination
+/tutorials/3d/faking_global_illumination.html,/tutorials/3d/faking_global_illumination.html
 /tutorials/3d/inverse_kinematics.html,/community/tutorials/3d/inverse_kinematics.html
+/tutorials/3d/reflection_probes.html,/tutorials/3d/reflection_probes.html
+/tutorials/3d/sdfgi.html,/tutorials/3d/using_sdfgi.html
+/tutorials/3d/using_lightmap_gi.html,/tutorials/3d/using_lightmap_gi.html
+/tutorials/3d/using_voxel_gi.html,/tutorials/3d/using_voxel_gi.html
```

Comment:
The source and destination URLs seem identical, is that expected?

Reply:
Oops, no, of course not. I'll fix it in a moment.

Reply:
Done!